src/typescript-reporter/reporter/TypeScriptReporter.ts

```diff
@@ -18,6 +18,13 @@ import {
 import { createPerformance } from '../../profile/Performance';
 import { connectTypeScriptPerformance } from '../profile/TypeScriptPerformance';
 
+// write this type as it's available only in the newest TypeScript versions (^4.1.0)
```

Comment:
So when typescript 4.1 ships we can lose this?

Reply:
We would have to upgrade typescript in dev dependencies, and then probably this type will be available (if it will be in the public API)